calico/networking/ipam/service-loadbalancer.mdx

```diff
@@ -0,0 +1,209 @@
+---
+description: Create LoadBalancer
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# LoadBalancer IP address management
```

Comment:
I've got questions for this page in a separate doc. Will follow up with you directly.